api/app/tests/apps/signals/factories.py

```diff
@@ -163,3 +166,43 @@ class Meta:
     @factory.post_generation
     def set_one_to_one_relation(self, create, extracted, **kwargs):
         self.signal = self._signal
+
+
+#
+# Category declarations
+#
+
+
```

Comment:
We should consider having a factory that produces valid categories, like we did for the locations. (Is nice for tests that interact with external systems - like the Sigmax integration.)

Reply:
The good part about having a data migration for the initial data population is that migrations are run to setup the test suite. So every test you write with Django `TestCase` will have a populated Category table!